packages/components/src/components/table/table/table.tsx

```diff
@@ -80,55 +66,13 @@ export class Table {
           </div>
         )}
         <div class="root">
```

Comment:
Do we still need this `div`?  All it does is have a `position: relative` style.

Reply:
Agree, it's not needed, removed it